be/src/runtime/sender_queue.cpp

```diff
@@ -661,6 +667,8 @@ Status DataStreamRecvr::PipelineSenderQueue::add_chunks(const PTransmitChunkPara
     DCHECK(!(keep_order && use_pass_through));
     DCHECK(request.chunks_size() > 0 || use_pass_through);
     if (_is_cancelled || _num_remaining_senders <= 0) {
+        LOG(WARNING) << print_id(request.finst_id()) << " adds chunks to "
```

Comment:
It's a normal case. prefer VLOG

Reply:
ok, I will change it to VLOG, as `_num_remaining_senders <= 0` is checked above.